combinetf2/fitter.py

```diff
@@ -19,9 +23,23 @@ def __init__(self, indata, options):
             raise Exception(
                 'option "--externalCovariance" only works with "--chisqFit"'
             )
-        if options.externalCovariance and options.binByBinStat:
+        if (
+            options.externalCovariance
+            and options.binByBinStat
+            and options.binByBinStat_type != "normal"
+        ):
             raise Exception(
-                'option "--binByBinStat" currently not supported for options "--externalCovariance"'
+                'option "--binByBinStat" only for options "--externalCovariance" with "--binByBinStat_type normal"'
+            )
+
+        if self.binByBinStat_type not in ["gamma", "normal"]:
```

Comment:
This can never happen because choices for the options are given, suggest to delete it

Reply:
It could if the fitter was used outside of combinetf2_fit and the arguments dictionary was constructed by hand